pkg/cmd/bazci/bazci.go

```diff
@@ -246,8 +246,11 @@ func (s *monitorBuildServer) handleBuildEvent(
 func (s *monitorBuildServer) Finalize() error {
 	if s.action == "build" {
 		for _, target := range s.builtTargets {
+			if target.OutputGroup == nil {
```

Comment:
nit: even if `target.OutputGroup` is nil (aka empty slice) we are good and still won't need this check right?